pkg/catalogv2/helm/helm2.go

```diff
@@ -36,10 +38,14 @@ var (
 	}
 )
 
+// isHelm2 checks if the value of the owner key of the received map is equal to TILLER.
+// Every helm2 release object contains this label.
 func isHelm2(labels map[string]string) bool {
 	return labels["OWNER"] == "TILLER"
 }
 
+// fromHelm2Data receives a helm3 release data.
```

Comment:
// fromHelm2Data receives a helm2 release data string of a helm chart installed.
// It then converts the string into Helm2 release struct and again to rancher
// v1.ReleaseSpec struct to return it.